test/unit/ngsiv2/amqpBinding-test.js

```diff
@@ -65,13 +65,11 @@ describe('AMQP Transport binding: measures', function() {
 
         nock.cleanAll();
 
-        // Note: /v1/updateContext response is not processed by IOTA so its content is irrelevant,
-        // as far as it is a 200 OK
         contextBrokerMock = nock('http://192.168.1.1:1026')
             .matchHeader('fiware-service', 'smartGondor')
             .matchHeader('fiware-servicepath', '/gardens')
-            .post('/v1/updateContext')
-            .reply(200, '{}');
+            .post('/v2/entities')
+            .reply(201);
```

Comment:
The right code is 204 (see http://telefonicaid.github.io/fiware-orion/api/v2/stable/)

It is not important (I guess that the IOTA agent looks for anything between 200 and 299) but better to use the right one in the tests ;)

Reply:
Wrong comment. NTC